src/Utilities/Constants.js

```diff
@@ -1,3 +1,2 @@
-
-export const SOURCES_API_BASE = `/r/insights/platform/topological-inventory/v0.0`;
-export const SERVICE_PORTAL_API_BASE = `/r/insights/platform/service-portal/v0.0`;
+export const SERVICE_PORTAL_API_BASE = process.env.BASE_PATH + '/service-portal/v0.0';
+export const TOPOLOGICAL_INVENTORY_API_BASE = process.env.BASE_PATH + '/topological-inventory/v0.0';
```

Comment:
Should we have environment variables for /service-portal/v0.0 and  /topological-inventory/v0.0?

Reply:
Maybe?  It's probably smart to have that and default it when not set.

Reply:
This should be:

onst base_path = process.env.BASE_PATH || '/r/insights/platform';

export const SERVICE_PORTAL_API_BASE = `${base_path}/service-portal/v0.0`;
export const TOPOLOGICAL_INVENTORY_API_BASE = `${base_path}/topological-inventory/v0.0`;

Reply:
I meant to put the default inside the parens...just a bug.